backend/libraries/types/src/chat_summary.rs

```diff
@@ -403,3 +403,9 @@ pub struct UpdatedRules {
 pub struct VideoCall {
     pub message_index: MessageIndex,
 }
+
+#[derive(CandidType, Serialize, Deserialize, Clone, Debug, Copy)]
+pub enum VideoCallType {
+    Broadcast,
```

Comment:
I would put `Default`, put the `#[default]` attribute on it, and derive `Default`.
This will allow the upgrade to work by populating all existing values with `VideoCallType::Default`.